### DISCUSSION TITLE
"connection not secure" but port is SSH

### DISCUSSION THREAD

**Discussion by buhtz** [scandum/tintin#172] [General]
I tried to connect to [`ff.mud.de`](http://finalfrontier.mud.de/start/spielen.html) using port `5555`. Based on the MUDs website this port should be secure. There is also an unsecure (telnet) port `7600`.

    #session ff ff.mud.de 5555

Gives me "Verbindung nicht sicher" (connection not secure).

Any idea about it?

**Comment by scandum** [comment]
Try:
```
#ssl ff ff.mud.de 5555
```
Keep in mind WinTin++ doesn't ship with ssl support, and it may require extra steps (having the gnutls library available) to get it included with the compilation on Linux.

**Reply by prool** [reply]
Hello

Excuse me:

I'm making compilation of tintin++ in Windows (cygwin).

Command '#ssl ff ff.mud.de 5555' is writes message:

#SSL: handshake failed error: No or insufficient priorities were set

Build in Ubuntu is working normally

With best regards, Serge Prool

**Reply by buhtz** [reply]
> Try:
> 
> ```
> #ssl ff ff.mud.de 5555
> ```

This works. Thanks.

**Comment by scandum** [comment]
Annoying, not sure if it's worth my time to look into. I mostly use WSL nowadays, and just use Cygwin 32 for the WinTin++ binary.

**Reply by prool** [reply]
Okay, dont' worry.

I'm using cygwin x64 (in Windows 7).

Current curious experiments looks:

tintin, running from cygwin shell - working with #ssl normally

tintin, running as Windows program from Windows shell (with some DLL files) - have #SSL error

All strange in strange world.

Let it be so.

With best regards, Prool
